libs/jwst-storage/src/storage/blobs/bucket_local_db.rs

```diff
@@ -145,7 +137,22 @@ impl BlobStorage<JwstStorageError> for BlobBucketDBStorage {
         id: String,
         params: Option<HashMap<String, String>>,
     ) -> JwstResult<BlobMetadata, JwstStorageError> {
-        todo!()
+        let workspace = get_workspace(workspace);
+        let key = build_key(workspace, id);
+
+        let metadata = self.bucket_storage.op.stat(&key).await?;
+
+        Ok(BlobMetadata {
```

Comment:
Hi, recommend only use db to query metadata, it's faster

Reply:
Fixed.